_episodes/45.md

```diff
@@ -0,0 +1,258 @@
+processes. DolphinScheduler enables users to create and manage successive jobs
+run easily, including support for different types of tasks, such as SQL
+statements, shell scripts, Spark, Kubernetes, and many others. In short, it's a
+powerful and user-friendly workflow orchestration platform that enables users to
+automate and manage their complex data processing tasks.
+
+Read [this blog on Trino and Apache DolphinScheduler](https://blog.devgenius.io/dolphinscheduler-helps-trino-quickly-realize-the-integrated-data-construction-of-lake-and-warehouse-cde095b6573b)
+to find out more.
+
+### Does DolphinScheduler have any computing engine or storage layer?
+
+DolphinScheduler is a powerful tool for managing and orchestrating data
+processing workflows across a range of computing engines and storage systems,
+but it does not provide its own computing or storage capabilities.
+
+## What are the differences with other workflow orchestration, such as Apache Airflow?
+
+Both Dolphin Scheduler and Airflow are designed to be scalable and highly
+available which supports large-scale distributed environments well.
+
+Airflow supports a wide range of third-party integrations, including popular
+data processing frameworks such as Trino, Spark, and Flink, as well as with
+cloud services such as AWS and GCP. Dolphin Scheduler supports a similar range
+of data processing frameworks and tools making both versatile platforms for
+managing diverse data processing tasks. Airflow is certainly ahead which makes
+it a more flexible solution for some workloads.
+
+Airflow focuses on workflow as code, meaning Airflow uses Python code to create
+and manage workflows. This limits the management and access to these workflows
+to platform engineers and doesn't enable the option for collaborating data
+management across domains. While there are third part tools that make it
+possible to create Airflow DAGs in a GUI, most of them are not up-to-date with
+the latest Airflow versions and not full-featured.
+
+DolphinScheduler project believes that future data governance belongs to a
+data civilization and should not be centralized to a single team.
+Product-focused engineering teams should have access to data and be able
+to orchestrate workflows without the need for extensive coding skills.
+DolphinScheduler uses a drag and drop web UI to create and manages workflows
+while also providing programmatic access using tools like Python SDK and Open
+API.
+
+A positive side effect of DolphinScheduler supporting users outside the data 
+team through a UI is that it offers robust security features. This includes
+authentication, authorization, and data encryption, to ensure that users' data
+and workflows are protected. Airflow has limited built-in security features,
+which may require additional configuration and customization to ensure that
+users' data and workflows are properly protected.
+
+Compared to some other workflow orchestration tools, DolphinScheduler has
```

Comment:
Don't compare .. just state it as a fact .. its a new project .. and they are working to improve docs and community support. 

So it can also be shorter